package.json

```diff
@@ -31,6 +34,7 @@
     "brunch": "^2.10.9",
     "chai": "^3.5.0",
     "eslint": "^3.15.0",
+    "istanbul": "jacksonrayhamilton/istanbul#09f9660",
```

Comment:
My fork of istanbul is off the last released version (0.4.5) and fixes [this bug](https://github.com/gotwarlost/istanbul/pull/816) which would otherwise occur when generating the HTML report.